runtime/sema/type.go

```diff
@@ -5440,6 +5560,10 @@ func (*CapabilityType) IsExternallyReturnable(_ map[*Member]bool) bool {
 	return true
 }
 
+func (t *CapabilityType) IsImportable(_ map[*Member]bool) bool {
+	return true
```

Comment:
Capabilities may not be imported! 

```suggestion
	return false
```

After this PR is merged, #712 should become easier to implement though 👍